app/jobs/attached_report_job.rb

```diff
@@ -10,7 +10,7 @@ def perform args
     user_id         = args.fetch :user_id
     organization_id = args.fetch :organization_id
 
-    report = model.where(id: ids).send method_name, options
+    report = model.where(id: ids.uniq).send method_name, options
```

Comment:
Esto en parte no importa mucho, después la DB devuelve siempre bien. Probá con algo como:

```ruby
User.where(id: [1, 1, 1]).count
```

Aunque si nos salvamos de que lo guarde en Redis, queda: https://youtu.be/IXnEs5mV-ak =)

Reply:
en ese caso prefiero meterlo antes de llegar acá.

Reply:
o no... jaja